opennms-model/src/main/java/org/opennms/netmgt/model/VertexInfo.java

```diff
@@ -0,0 +1,111 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2018 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2018 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.netmgt.model;
+
+import java.util.Objects;
+
+import org.opennms.netmgt.model.monitoringLocations.OnmsMonitoringLocation;
+
+/** Simplified OnmsNode. We use it as a view on OnmsNode that has less attributes and no lazy loading */
+public class VertexInfo {
```

Comment:
`VertexInfo`  seems pretty abstract since a vertex in the context of the topology can be just about anything.

Maybe `NodeVertexInfo` would be more appropriate here?

Reply:
@j-white, agreed. I think we should have a common name for these objects (currently: VertexInfo, CdpLinkInfo, later all [Lldp|Isis|...]LinkInfo). They are lightweight entities that contain no references and have only the necessary  attributes to construct the topology. How do you like: NodeTopologyEntity, CdpTopologyEntity, IsisTopologyEntity,...

That would be consistent with @mvrueden remark (calling the cache: TopologyEntityCache)

Reply:
I renamed them to NodeTopologyEntity and CdpLinkTopologyEntity